extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/runtime/RedisClientUtil.java

```diff
@@ -59,9 +67,51 @@ public static RedisOptions buildOptions(RedisConfiguration redisConfig) {
             options.setUseReplicas(redisConfig.replicas.get());
         }
 
+        if (redisConfig.password.isPresent()) {
+            options.setPassword(redisConfig.password.get());
+        }
+
+        options.setNetClientOptions(toNetClientOptions(redisConfig));
+
         return options;
     }
 
+    private static NetClientOptions toNetClientOptions(RedisConfiguration redisConfig) {
+        NetClientOptions netClientOptions = new NetClientOptions()
+                .setTcpKeepAlive(true)
+                .setTcpNoDelay(true);
```

Comment:
Do we think these two can be made configurable too?

Reply:
Yes, it should be correct.

Reply:
Thanks, they can take a default value of `true`